util.go

```diff
@@ -382,3 +382,35 @@ func MeasureSeedRPCServerContext(ctx context.Context, seedRPCList *nkngomobile.S
 
 	return nkngomobile.NewStringArray(rpcAddrs...), nil
 }
+
+func ResolveDest(dest string, resolver []ResolverInterface) (string, error) {
+	var err error
+	for _, r := range resolver {
+		var d string
+		d, err = r.Resolve(dest)
+		if err == nil {
```

Comment:
I think we should return err if `err != nil` because it means network error, or invalid address, that should be handled by developers.
Basically we should use `len(d)` (when `err == nil`) to determine if address is resolved